discovery-provider/src/app.py

```diff
@@ -498,6 +496,16 @@ def configure_celery(celery, test_config=None):
 
     logger.info("Redis instance initialized!")
 
+    # Start main indexer
+    celery.send_task("index_nethermind")
```

Comment:
should send_task come after finalize?

Reply:
Easy enough change. we had another send_task earlier up, but I assume that will get removed at some point as it's a backfill